src/colorizer/CanvasWithOverlay.ts

```diff
@@ -1,104 +1,102 @@
 import { Vector2 } from "three";
```

Comment:
Lots of things changed in CanvasWithOverlay! 
1. All the "option" types have been renamed to "style" to reflect their purpose.
2. The style types have been moved to `canvas/types.ts.`
3. All rendering logic for the scale bar and timestamp has been moved into `canvas/elements`.